cluster/charts/universal-crossplane/values.yaml.tmpl

```diff
@@ -29,9 +29,6 @@ serviceAccount:
 leaderElection: true
 args: []
 
-# How frequently Crossplane may reconcile its resources (seconds). Default: 10
-maxReconcileRate: ""
```

Comment:
This is a parameter added but reverted later: https://github.com/crossplane/crossplane/pull/3151

There is no such parameter in the xp chart anymore.